geoips/interface_modules/sectors_dynamic/tc_1024x1024/tc_1km_1024x1024.yaml

```diff
@@ -0,0 +1,15 @@
+interface: "sectors_dynamic"
+metadata:
+  name: "tc_1km_1024x1024"
+  family: "standard"
+  short-description: "The tc_1km_1024x1024 boundaries configuration"
+  long-description: "Using pixel_width: 1000, pixel_height: 1000, num_lines: 1024, num_samples: 1024."
+spec:
+  area_def_generator:
```

Comment:
Note I updated the YAML files to be more consistent with how we handled "product_templates" (ie, specify the interface, then name/arguments that specify how to call the specific plugin).

Reply:
Okay, understood, We talked about this on slack and I understand now.